src/sdltiles.cpp

```diff
@@ -1050,8 +1051,6 @@ void cata_tiles::draw_om( const point &dest, const tripoint_abs_omt &center_abs_
             const int name_length = name.length();
             const point draw_pos = abs_sm_to_draw_label( pos, name_length );
             SDL_Rect clipRect = { draw_pos.x, draw_pos.y, name_length * fontwidth, fontheight };
-            printErrorIf( SDL_RenderSetClipRect( renderer.get(), &clipRect ) != 0,
```

Comment:
Since this is removed, the comment above about drawing labels last can be removed.

Reply:
Done.